WordPress/src/main/res/values/strings.xml

```diff
@@ -1340,6 +1351,9 @@
     <string name="reader_btn_follow">Follow</string>
     <string name="reader_btn_unfollow">Following</string>
 
+    <string name="reader_followed_blog_notifications">Following &lt;b&gt;%1$s&lt;/b&gt;&lt;br&gt;Enable post notifications?</string>
```

Comment:
That looks tricky to translate! Might be simpler to use `Following %s\nEnable post notifications?` and have the app insert the HTML bolding in the `%s` replacement.